docs/source/quick_start/first_run.rst

```diff
@@ -0,0 +1,175 @@
+
+First Run
+=========
+
+Now you have installed MCDR, it's time to run it for the first time
+
+.. note::
+
+    Most common use of MCDR is to control a Minecraft server, so we also take this as the demonstrated use case here
+
+.. tip::
+
+    To manage Minecraft server with MCDR, It is recommended to have basic knowledge of common Minecraft server softwares
+    
+    Which means, you should know:
+
+    * The Minecraft server softwares: what are they, how to configure (one of) them
+    * Start command of Minecraft server: what is it, what its arguments mean
+    * Console of the server: what it prints, what command can you send to it
+    * YAML and JSON files: what are they, how to edit them
+    
+    If you have not, Google is your friend
+
+Prepare
+-------
+
+To manage a Minecraft server with MCDR, you should have a **well-configured** Minecraft server software
+
+For example, this is a typical directory structure of vanilla Minecraft server:
+
+.. code-block:: text
+
+    server/
+    ├── libraries/
+    ├── logs/
+    ├── versions/
+    ├── world/
+    ├── banned-ips.json
+    ├── banned-players.json
+    ├── eula.txt
+    ├── ops.json
+    ├── server.jar
+    ├── server.properties
+    ├── usercache.json
+    └── whitelist.json
+
+With this command to start it:
+
+.. code-block:: bash
+
+    java -Xms1G -Xmx2G -jar minecraft_server.jar nogui
+
+You can also use other server software, such as Fabric, Spigot, Paper, etc.
+
+.. tip::
+
+    Configure and troubleshoot your server software **BEFORE** you introduce MCDR. Otherwise, you can't distinguish whether a problem is caused by MCDR or your server software
+
+    MCDR was born as a daemon for common server softwares of Minecraft: Java Edition. Most plugins are predicated on that. If you want to run MCDR with a Bedrock Edition server, consider using Java Edition server software with `GeyserMC <https://geysermc.org/>`__ to provide Bedrock Edition compatibility
+
+    Of course, if you don't need the MCDR plugin community, you can run anything you want with MCDR
+
+Initialize
+----------
+
+After MCDR has been installed, you can verify the installation with the following command:
+
+.. code-block-mcdr-version:: bash
+
+    $ mcdreforged
+    MCDReforged v@@MCDR_VERSION@@
+
+Let's say your are going to start MCDR in a folder named ``my_mcdr_server``. Then you can run the following commands to initialize the environment for MCDR:
+
+.. code-block:: bash
+
+    cd my_mcdr_server
+    mcdreforged init
+
+MCDR will generate its default structure like this:
+
+.. code-block::
+
+    my_mcdr_server/
+     ├─ config/
+     ├─ logs/
+     │   └─ MCDR.log
+     ├─ plugins/
+     ├─ server/
+     ├─ config.yml
+     └─ permission.yml
+
+Animated demo:
+
+.. asciinema:: resources/init.cast
```

Comment:
What shell environment has the given behavior that will echo the name of the `cd`-ed directory

```
~ $ cd my_mcdr_server/                                                                                                  
my_mcdr_server $ mcdreforged init 
```